projects/core/src/lib/download-vector-layer/download-vector-layer.component.html

```diff
@@ -0,0 +1,71 @@
+<geoengine-sidenav-header>Download Vector Layer</geoengine-sidenav-header>
+
+<form [formGroup]="form">
+    <mat-dialog-content>
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Area</mat-card-title>
+                <mat-card-subtitle>Specify a rectangular region for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <div fxLayout="row">
+                    <mat-form-field appearance="fill">
+                        <mat-label>min X</mat-label>
+                        <input matInput type="number" formControlName="bboxMinX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max X</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>min Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMinY" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxY" />
+                    </mat-form-field>
```

Comment:
![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/4ad0fd80-e240-448d-b68f-16c1f9f3ab59)

Could we have a small gap, e.g. 0.5rem, between these inputs?
Maybe we should let them break into a new line, e.g. 2x2, if the fields become too small.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/722/commits/86e887b012a8497b64c2c6adb00a2b8ca01d9dac